### DISCUSSION TITLE
Help Needed with Gemini

### DISCUSSION THREAD

**Discussion by TyrealQ** [lencx/Noi#85] [Q&A]
Noi Ask doesn't work with Gemini but works fine with ChatGPT and Claude. Would appreciate any help to fix this.

**Comment by lencx** [comment]
https://github.com/lencx/Noi/issues/7#issuecomment-1937769968

**Reply by TyrealQ** [reply]
Thanks!

**Comment by TyrealQ** [comment]
> Noi Ask doesn't work with Gemini but works fine with ChatGPT and Claude. Would appreciate any help to fix this.

Noi Ask doesn't work with ChatGPT since switching to team version but works OK with Gemini and Claude. Any advice on fixing this? Thanks!
